pkgs/stdenv/generic/check-meta.nix

```
@@ -16,8 +16,8 @@ let
   allowUnfree = config.allowUnfree or false
     || builtins.getEnv "NIXPKGS_ALLOW_UNFREE" == "1";
 
-  whitelist = config.whitelistedLicenses or [];
-  blacklist = config.blacklistedLicenses or [];
+  allowlist = config.allowlistedLicenses or [];
+  blocklist = config.blocklistedLicenses or [];
```

What about providing backwards compatibility via:

```suggestion
  allowlist = config.allowlistedLicenses or config.whitelistedLicenses or [];
  blocklist = config.blocklistedLicenses or config.blacklistedLicenses or [];
```

---

I had this but deleted it, but if u think it's fine let's do it :smile: